metrics/flake8_high_water_mark

```diff
@@ -1 +1 @@
-166
+9
```

Comment:
I don't exactly understand how these are used but why is this such a large numerical change?

Reply:
So these "high water marks" are the number of violations there were of a particular protocol. In this case, we had 166 Flake8 violations and brought them down to 9. I mostly did this through using `autopep8` then hand correcting some of the other errors. We still have 9 flake8 errors remaining... but those are from "too high complexity" which is more than I wanted to take on in a single PR.